lab8V1.py

```diff
@@ -0,0 +1,50 @@
+from random import randint
+
+
+def fill_the_matrix(x, y):
+    mat = []
+    for i in range(x):
+        row = [randint(-9, 10) for _ in range(y)]
+        mat.append(row)
+    return mat
```

Comment:
if you've already known the comprehensions, then use nested compr. and rewrite it without append()